routes/dashboard/portals/project-allotment-student/index.js

```diff
@@ -13,13 +13,32 @@ var adminsModel = fq("schemas/admins");
 var projectsModel = fq("schemas/projects");
 var applicationsModel = fq("schemas/project-applications");
 
+var forbidden_batches = [];
+settingsModel.find(
```

Comment:
Shift this to middleware in `router.use()` as a server restart would be required in this case whenever the batches setting is modified from the admin portal.